static/app/gettingStartedDocs/python/python.tsx

```diff
@@ -72,8 +80,36 @@ sentry_sdk.init(
 ];
 // Configuration End
 
-export function GettingStartedWithPython({dsn, ...props}: ModuleProps) {
-  return <Layout steps={steps({dsn})} {...props} />;
+export function GettingStartedWithPython({
+  dsn,
+  activeProductSelection = [],
+  ...props
+}: ModuleProps) {
+  const otherConfigs: string[] = [];
+
+  let sentryInitContent: string[] = [
+    `  dsn="${dsn}",`,
+    `  integrations=[DjangoIntegration()],`,
```

Comment:
I think this `DjangoIntegration` is copy-paste leftover. Sorry I didn't catch this in my initial review.